code/integration-tests/local-integration-tests/src/cross_chain_transfer.rs

```diff
@@ -292,7 +291,7 @@ fn this_native_transferred_from_sibling_to_native_is_not_enough() {
 				)
 				.into()
 			),
-			399_600_000_000
+			Limited(399_600_000_000)
```

Comment:
Can we not use the same constant `DEFAULT_SENDER_WEIGHT_LIMIT` ?

Reply:
sure. just started replacing.